pkg/http/server_test.go

```diff
@@ -97,3 +97,66 @@ func TestNewServer(t *testing.T) {
 		}
 	}
 }
+
+var tlsCipherSuitesTests = []struct {
+	ciphers    []uint16
+	shouldFail bool
+}{
+	{ciphers: nil, shouldFail: false},
+	{ciphers: defaultCipherSuites, shouldFail: false},
+	{
+		ciphers: []uint16{ // unsupported cipher suites
```

Comment:
init a separate variable with this value and use it here. e.g. `unsupportedCiphers = []uint16{...}` then `{unsupportedCiphers, true}`

Reply:
Yes, that's a good idea 👍